p2d/parse_polygon_package.py

```diff
@@ -27,7 +27,7 @@ def parse_samples_explanations(notes):
                 exit(1)
             assert(test_id != -1)
             assert(test_id not in explanations)
-            curr = curr[0].upper() + curr[1:]  # Capitalize first letter.
+            curr = curr[0].upper() + curr[1:]   # Capitalize first letter.
```

Comment:
Unless there's a reason, please undo this change.

Reply:
Nope, it was accidental, I'll undo it.